src/main/java/com/epam/jconference/service/UserService.java

```diff
@@ -0,0 +1,14 @@
+package com.epam.jconference.service;
+
+import com.epam.jconference.controller.dto.UserDto;
+import com.epam.jconference.service.mapper.UserMapper;
+import com.epam.jconference.service.model.User;
+import org.springframework.stereotype.Service;
+
+@Service
```

Comment:
It has some missunderstanding for me, because UserService should be the interface which defines all methods there for example, and after that we are using UserServiceImpl class which implements all methods from UserService, and this class should be annotated with @Service annotation

Reply:
It was fixed. Please, check my fresh commit.

Reply:
Looks good now. Thanks!